doc/releasenotes.rst

```diff
@@ -3,9 +3,9 @@
 *************
 Release Notes
 *************
-0.7.0
+0.8.0
```

Comment:
Oops, we should just be adding new details on top - not removing old details.  I also see now that 0.7.1 and. 0.7.2 notes are missing.